<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TagTool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnSetColumn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string extracomment="Sets the column with text to be searched. Select a column below and click this button to set it."/>
        </property>
        <property name="text">
         <string>Set Active Column</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTagAll">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string extracomment="Tags all rows with a single column (Keyword). All keywords will appear here seperated by a comma."/>
        </property>
        <property name="text">
         <string>Tag</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_save">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="txtActiveStat">
        <property name="text">
         <string>No Active Column selected</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="act_open"/>
    <addaction name="separator"/>
    <addaction name="act_quit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="act_editor"/>
    <addaction name="act_clearTable"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="act_about"/>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="menu_openFile"/>
   <addaction name="separator"/>
   <addaction name="menu_keywordEditor"/>
   <addaction name="separator"/>
   <addaction name="menu_clearTable"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="act_open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="act_quit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="act_editor">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Keyword Editor</string>
   </property>
  </action>
  <action name="act_clearTable">
   <property name="text">
    <string>Clear Table</string>
   </property>
  </action>
  <action name="act_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="menu_openFile">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="menu_keywordEditor">
   <property name="text">
    <string>Keyword Editor</string>
   </property>
   <property name="toolTip">
    <string>Keyword Editor</string>
   </property>
  </action>
  <action name="menu_clearTable">
   <property name="text">
    <string>Clear Table</string>
   </property>
   <property name="toolTip">
    <string>Clear table</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
